### PR TITLE
Tried to clarify that Bundle and AbstractBundle are different

### PR DESCRIPTION
**Pull request by Andreas-Schoenefeldt**
Please see https://stackoverflow.com/questions/74704768/invalidconfigurationexception-with-custom-bundle-in-symfony6-2 for the details.

### CODE REVIEW

File: bundles/configuration.rst [hunk 1]
```diff
@@ -319,12 +319,16 @@ In your extension, you can load this and dynamically set its arguments::
...
+Using the AbstractBundle Class
+------------------------------
 
 .. versionadded:: 6.1
 
     The ``AbstractBundle`` class was introduced in Symfony 6.1.
+    
+.. caution::
+
+    This is an alternative implementation to the above mentioned Bundle class. Bundle and AbstractBundle are not compatible.
```
Comment (yceruto) [inline bundles/configuration.rst:331]: I'd say after the first sentence something like...
> You can use one of these approaches but not both at the same time.

that would make clearer what "not compatible" means in this context.

**Review by yceruto — COMMENTED** [review]
This improvement should target 6.1 branch. Thanks!

File: bundles/configuration.rst (hunk 1 above)
Comment (Andreas-Schoenefeldt) [reply bundles/configuration.rst:331]: Thx, I'm definitely not a native speaker :)

**Comment by Andreas-Schoenefeldt** [comment]
I'll open a new one against 6.1

**Comment by Andreas-Schoenefeldt** [comment]
https://github.com/symfony/symfony-docs/pull/17536 Thx again :)

**Pull request closed without merging**